src/main/java/link/infra/packwiz/installer/bootstrap/Bootstrap.java

```diff
@@ -287,14 +273,23 @@ private static Release requestRelease() throws IOException, GithubException {
 		return rel;
 	}
 
+	private static String getAssetUrl(String property, JsonObject asset) throws GithubException {
+		return getStringProperty(property, asset, "Asset Download URL property");
+	}
+	
+	private static String getStringProperty(String property, JsonObject obj, String displayName) throws GithubException {
+		JsonValue value = obj.get(property);
+		if (value == null || !value.isString()) {
+			throw new GithubException("$displayName (" + property + ") cannot be found");
```

Comment:
Java doesn't have string templating :P

```suggestion
			throw new GithubException(displayName + " (" + property + ") cannot be found");
```